api/internal/handler/data_loader/route_import.go

```diff
@@ -113,7 +113,6 @@ func (h *ImportHandler) Import(c droplet.Context) (interface{}, error) {
 			MergeMethod: input.MergeMethod == "true",
 			TaskName:    input.TaskName,
 		}
-		break
```

Comment:
why change this?

Reply:
The go-lint advised me to change this. I just followed it.

Reply:
I will change it back in next commit.